client/tests/integration/multisignature_account.rs

```diff
@@ -17,24 +17,20 @@ fn transaction_signed_by_new_signatory_of_account_should_pass() -> Result<()> {
     // Given
     let account_id: AccountId = "alice@wonderland".parse().expect("Valid");
     let asset_definition_id: AssetDefinitionId = "xor#wonderland".parse().expect("Valid");
-    let create_asset = RegisterExpr::new(AssetDefinition::quantity(asset_definition_id.clone()));
+    let create_asset =
+        RegisterBox::asset_definition(AssetDefinition::quantity(asset_definition_id.clone()));
```

Comment:
```suggestion
        RegisterBox::asset_definition(AssetType::Quantity, asset_definition_id.clone());
```

I think this makes a better API

Reply:
Well initially with `*Box` isi constructors I wanted to reduce boilerplate with sub-boxes.

In that particular case it's done for uniformity. Not sure we should take parameters out of `AssetDefinition` and place them into `RegisterBox`. Kind of abstraction break